noir/noir-repo/acvm-repo/brillig_vm/src/black_box.rs

```diff
@@ -143,6 +143,19 @@ pub(crate) fn evaluate_black_box<Solver: BlackBoxFunctionSolver>(
             memory.write_slice(memory.read_ref(result.pointer), &[x.into(), y.into()]);
             Ok(())
         }
+        BlackBoxOp::VariableBaseScalarMul { point_x, point_y, scalar_low, scalar_high, result } => {
+            let point_x = memory.read(*point_x).try_into().unwrap();
+            let point_y = memory.read(*point_y).try_into().unwrap();
+            let scalar_low = memory.read(*scalar_low).try_into().unwrap();
+            let scalar_high = memory.read(*scalar_high).try_into().unwrap();
+            let (out_point_x, out_point_y) =
+                solver.variable_base_scalar_mul(&point_x, &point_y, &scalar_low, &scalar_high)?;
+            memory.write_slice(
+                memory.read_ref(result.pointer),
+                &[point_x.into(), point_y.into(), out_point_x.into(), out_point_y.into()],
```

Comment:
```suggestion
                &[out_point_x.into(), out_point_y.into()],
```
Looks like we are writing the original point in the result slots